torch/csrc/autograd/engine.cpp

```diff
@@ -933,6 +935,21 @@ auto Engine::execute(const edge_list& roots,
   // in dist_engine.cpp).
   auto& fut = graph_task->future_result_;
   fut->wait();
+
+  // Syncs leaf streams with ambient (current) streams of the thread that called execute().
+  // See note "Streaming backwards". leaf_streams is an unordered_set so it should only
```

Comment:
Which note do you refer to here? If it is in the website doc, you can add a link, otherwise we use [] to mark the note name and make it more grepable

Reply:
Changed to Note [Streaming backward]. The Note gives a bit more low level description that isn't in the sphinx.